selene_sdk/samplers/random_positions_sampler.py

```diff
@@ -122,12 +128,14 @@ class RandomPositionsSampler(OnlineSampler):
     mode : str
         The current mode that the sampler is running in. Must be one of
         the modes listed in `modes`.
-
     """
     def __init__(self,
                  reference_sequence,
                  target_path,
                  features,
+                 train_size,
```

Comment:
change this to N_train, N_validation, N_test, etc. and `N_test` needs to be an optional argument